### PR TITLE
corrections done

### LINKED ISSUE #47: Spelling error correction navbar admin
stage => stages client => clients

### PR DESCRIPTION
**Pull request by bouboumael**

### CODE REVIEW

**Review by Loic-Code — DISMISSED** [review]

File: src/View/Admin/Includes/_navbar.html.twig [hunk 1]
```diff
@@ -7,8 +7,8 @@
 		<div class="collapse navbar-collapse" id="navbarNavAltMarkup">
 			<div class="navbar-nav">
 				<a class="btn-nav" href="/Admin/course">Cours</a>
-				<a class="btn-nav" href="">Stage</a>
-				<a class="btn-nav" href="">Client</a>
+				<a class="btn-nav" href="">Stages</a>
+				<a class="btn-nav" href="">Clients</a>
```
Comment (LucyMerlier) [inline src/View/Admin/Includes/_navbar.html.twig]: "btn-nav" => is it a bootstrap class?

**Review by LucyMerlier — COMMENTED** [review]
"btn-nav" => is it a bootstrap class?

File: src/View/Admin/Includes/_navbar.html.twig (hunk 1 above)
Comment (bouboumael) [reply src/View/Admin/Includes/_navbar.html.twig:11]: yes

Comment (Loic-Code) [reply src/View/Admin/Includes/_navbar.html.twig:11]: yes

Comment (bouboumael) [reply src/View/Admin/Includes/_navbar.html.twig]: correction done

**Review by Loic-Code — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**